app/interfaces/api/entities/ccr/offence.rb

```diff
@@ -3,7 +3,9 @@ module Entities
     module CCR
       class Offence < API::Entities::BaseEntity
         expose :unique_code
-        expose :offence_class, using: API::Entities::CCR::OffenceClass
+        expose :offence_class,
+               if: ->(instance, _opts) { instance.fee_schemes.first.number.eql?(9) },
```

Comment:
**[not a blocker/tiny nit pick]** If `9` means something here, I would consider setting it in some constant `AGFS_SCHEME9` (maybe in the `FeeScheme` class) so it's a bit more clear what it is and can be re-used across different places.

Reply:
in terms of helpers and clarity it would be nice to have some delegation allowing `instance.for_agfs_scheme?(9)` or for_scheme?(AGFS_SCHEME9)` - but this can be iterated on.